src/Sylius/Component/Translation/Model/AbstractTranslatable.php

```diff
@@ -19,15 +19,11 @@
 abstract class AbstractTranslatable implements TranslatableInterface
 {
     /**
-     * Translations.
-     *
      * @var TranslationInterface[]
```

Comment:
IMHO putting `Collection` here is a bad idea as we don't want to couple this interface with Doctrine Commons